gen_verilog_sim_multicore.py

```
@@ -0,0 +1,52 @@
+
+if __name__ == '__main__':
+    parser = ArgumentParser()
+    parser.add_argument('imem', help = 'Absolute path to imem file')
+    parser.add_argument('dmem', help = 'Absolute path to dmem file')
+    args = parser.parse_args()
+
+    if isdir(OUT):
+        rmtree(OUT)
+
+    try:
+        run(
```

Ruff (0.14.2)

`subprocess` call with `shell=True` identified, security issue

(S602)